docs/config/README.md

```diff
@@ -528,6 +529,35 @@ symbol = "+ "
 threshold = 4
 ```
 
+
+## Kubernetes
+
+Displays the current Kubernetes context name and, if set, the namespace from
+the kubeconfig file. The namespace needs to be set in the kubeconfig file, this
+can be done via `kubectl config set-context starship-cluster --namespace
+astronaut`. If the `$KUBECONFIG` env var is set the module will use that if
+not it will use the `~/.kube/config`.
+
+### Options
+
+| Variable   | Default       | Description                                         |
```

Comment:
Total nitpick:

The "Options" tables usually end with style and disabled for consistency.
Could you please reorder them?